chromium_src/chrome/browser/ui/commander/window_command_source.cc

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/chromium_src/chrome/browser/ui/commander/window_command_source.h"
+
+#include "src/chrome/browser/ui/commander/window_command_source.cc"  // IWYU pragma: export
```

Comment:
`// IWYU pragma: export` should be added only to headers.

Reply:
Cool!